packages/scandipwa/src/component/CheckoutShipping/CheckoutShipping.container.js

```diff
@@ -105,27 +106,17 @@ export class CheckoutShippingContainer extends PureComponent {
         };
     }
 
-    componentDidUpdate(prevProps) {
-        const { shippingMethods: prevShippingMethods } = prevProps;
-        const { shippingMethods } = this.props;
+    componentDidUpdate() {
+        const { isPickInStoreMethodSelected, shippingMethods } = this.props;
+        const pickupMethod = shippingMethods.find((el) => el.carrier_code === 'instore');
```

Comment:
Please move `instore` to the constant and put it to the `CheckoutShipping.config.js` file.

Reply:
Replaced it with `STORE_IN_PICK_UP_ATTRIBUTE_CODE` from `StoreInPickUp.config`